store/src/java/com/zimbra/cs/imap/ImapRequest.java

```diff
@@ -610,6 +614,51 @@ String readSequence() throws ImapParseException {
         return validateSequence(readContent(SEQUENCE_CHARS), true);
     }
 
+    CacheEntryType[] readCacheEntryTypes() throws IOException, ImapParseException {
+        String[] cacheStrTypes = readAstring().split(",");
```

Comment:
nit: Would it make more sense (be more IMAP like) to not use a CSV and instead use a parenthesized list?
- () :: default to all
- (cache1 cache2 ...)